AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_constants.py

```diff
@@ -52,6 +54,7 @@
 """
 General constants
 """
+TITLE_STRING = "Title"
```

Comment:
Can we give a slightly more meaningful name here? Is this the SC node title we're mapping to a const here or something else? Let's make sure the name reflects the intent for easier readability.

Reply:
elaborated the name of the variable. also moved it to a section dedicated to node inspector.